packages/gestalt/src/Button.jsdom.test.js

```diff
@@ -13,10 +13,21 @@ describe('Button', () => {
     expect(mockOnClick).toHaveBeenCalled();
   });
 
-  it('forwards a ref to the innermost button element', () => {
-    const ref = React.createRef();
+  it('forwards a ref to the innermost button element with type button', () => {
```

Comment:
Could we add a test for a disabled Button with `href`?

Reply:
DONE